core/server/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -946,8 +961,10 @@ private TraversalResult traverseToInode(String[] pathComponents, LockMode lockMo
         if (pathComponents[0].equals("")) {
           if (getLockModeForComponent(0, pathComponents.length, lockMode, lockHints)
               == LockMode.READ) {
+            // Do not need to check parent for root.
```

Comment:
Sure, the conditions in which to use `lockRead/Write` are documented there so it should not be confusing.